Gemfile

```diff
@@ -22,7 +22,7 @@ end
 # as a dependency of govuk_content_models
 gem 'mongo', '>= 1.7.1'
 
-gem 'gds-sso', '10.0.0'
+gem 'gds-sso', '11.0.0'
```

Comment:
gds-sso here is used to authenticate requests for draft versions of content from private-frontend.

gds-sso 11 adds support for the organisation_content_id field on the user model.  This has not been added to the user model in govuk_content_models yet.